.github/workflows/build.yml

```diff
@@ -131,7 +132,7 @@ jobs:
       working-directory: ${{ matrix.vcpkg_path }}
 
     - name: Build packages
-      run: ./vcpkg install --vcpkg-root=${{ matrix.vcpkg_path }} --clean-after-build --recurse ${{ env.VCPKG_PACKAGES }}
+      run: ./vcpkg install --vcpkg-root=${{ matrix.vcpkg_path }} --x-buildtrees-root=${{ matrix.vcpkg_buildtrees-root }} --clean-after-build --recurse ${{ env.VCPKG_PACKAGES }}
```

Comment:
Why is this only needed for 2.5? Should it be backported to 2.4?

Reply:
Because Qt6 uses onger paths.